### DISCUSSION TITLE
WHMCS and Blesta support

### DISCUSSION THREAD

**Discussion by ianlot** [pelican-dev/panel#59] [Ideas]
a better plugin than the one that exist for Pterodactyl, with multi port support: if a game requires more than 1 port. For example MTA:SA needs 3 port 220013, 22126, 22005.

**Comment by lancepioch** [comment]
There will be a new plugin(s) repo in the future that you can submit these ideas to!

**Comment by hornethosting** [comment]
ptero bill will probly work still

**Reply by ianlot** [reply]
That's great, but some people want to use it with WHMCS.

**Reply by hornethosting** [reply]
thats what ptero bill is its a whmcs plugin for  Pterodactyl

**Reply by ianlot** [reply]
> thats what ptero bill is its a whmcs plugin for  Pterodactyl

You are right, I mistook it for something else. But it is paid, not free, like this https://github.com/pterodactyl/whmcs
